pkgs/CyberAgent/reminder-lint/registry.yaml

```diff
@@ -13,8 +13,11 @@ packages:
         replacements:
           amd64: x86_64
           darwin: apple-darwin
-          linux: unknown-linux-musl
+          linux: unknown-linux-gnu
```

Comment:
Why did you change `musl` to `gnu`?

Reply:
Sorry! It didn't work properly in the scaffolded registry, so I was trying to change the settings. These are the remnants of that.

I fixed this line in the following commit:
https://github.com/aquaproj/aqua-registry/pull/41841/commits/e3daffe980dd80835a4389b45a9c8a52acdc6ba5